packages/manager/modules/support/src/new-ticket/creation-form/creation-form.controller.js

```diff
@@ -1,17 +1,39 @@
 import findLast from 'lodash/findLast';
 
 export default class SupportNewCreationFormController {
+  /* @ngInject */
+  constructor(SupportNewTicketService) {
+    this.SupportNewTicketService = SupportNewTicketService;
+  }
+
+  $onInit() {
+    return this.SupportNewTicketService.getSupportLevel().then((level) => {
+      this.supportLevel = level;
+      if (this.shouldSelectUrgency()) {
+        return this.SupportNewTicketService.getUrgencies().then((urgencies) => {
+          this.urgencies = urgencies;
+        });
+      }
+      return level;
+    });
+  }
+
   get subject() {
     const issue = findLast(this.issues, 'subject');
     return issue ? issue.subject : undefined;
   }
 
+  shouldSelectUrgency() {
+    return ['business', 'enterprise'].indexOf(this.supportLevel) >= 0;
```

Comment:
Wouldn't it be better to have capabilities for each support level ?

````js
class SupportLevel {
   shouldSelectUrgency() {
        return includes(
            ['business', 'enterprise'],
            this.supportLevel,
        );
    }
}
````

this way capabilities and duties of each support level would be at the same place :)